### DISCUSSION TITLE
Fluent V9: TreeItem disabled state

### DISCUSSION THREAD

**Discussion by nmann4** [microsoft/fluentui#30723] [General] [answered]
Hi is there a simple way to disable a TreeItem? I have to implement this for a feature I'm working on and would make my life easier. I could be missing something but didn't see it in the code. Happy to file a feature request or maybe there is reasons for this.

Thanks!

**Comment by behowell** [comment, marked as answer]
No, TreeItem doesn't support a disabled state.  You could add your own css class/styles to the TreeItem to make it appear disabled. If needed, you can also control its `open` state to prevent it from being opened while disabled (see the [Tree: Open Item Controlled](https://react.fluentui.dev/?path=/docs/components-tree--default#open-item-controlled) example on the documentation site).

**Reply by nmann4** [reply]
Thanks for the quick reply. I was hoping for an easy way out but I will work around it. 👍
